### DISCUSSION TITLE
How to update sqlite3 in middle of code ?(python)

### DISCUSSION THREAD

**Discussion by 3ekodomo** [community/community#68046] [Programming Help] [answered]
### Body

i am want to run pip shell command in code, without using a shell. I want to update sqlite3 to latest version without. As i don't have access to shell. Guide me please.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by heartlog** [comment]
Are you using vps/hosting or something similar to that ?

**Comment by 3ekodomo** [comment]
Yea... Just want to use pip to do that. But don't have access to shell.

**Comment by heartlog** [comment, marked as answer]
You can use subprocess to do this. (It's native python package. So, **you don't need to pip install it.**
## Code
```py
import subprocess

# Uninstall old version
subprocess.run(['pip', 'uninstall', 'sqlite3', '-y'])

# Install latest version
subprocess.run(['pip', 'install', 'sqlite3'])

print("SQLite updated successfully.")
```

### Remember
if you're using website or provider which is limited python with restricted docker image.(Like no pip install allowed)

**Comment by 3ekodomo** [comment]
Thanks a lot 👍 its working.

**Reply by heartlog** [reply]
You're welcome 🤗
